### DISCUSSION TITLE
Edit of QSO after LOTW Upload? (Also Future QSOs!)

### DISCUSSION THREAD

**Discussion by petebrunelli** [magicbug/Cloudlog#1775] [General]
Hi folks,
I had entered the incorrect date for several QSOs and they were uploaded to LOTW before I found the error. Will the edited files get re-uploaded or do I need to manually upload them to LOTW?

Also, the QSO date was in the future. I don't need a lot of guardrails but a popup would have helped ;)

Thanks, Pete N1QDQ

**Comment by phl0** [comment]
Well, at least you can edit the QSOs, set LotW uploaded to `no` and they will get re-uploaded on the next LotW sync.

**Reply by petebrunelli** [reply]
That's brilliant. Thanks for explaining that. I made that change. I'm still getting my head around Cloudlog after a few years of running a different logging app. So far Cloudlog is miles ahead. Very happy.

**Reply by petebrunelli** [reply]
That worked perfectly. Thanks again!

**Comment by phl0** [comment]
I made a basic check for the QSO date to not be in the future. But it needs further testing especially with date format and timezones. The basics are here:

[61dbf5595c93a89ade1770a4107b68210a3c30d5](https://github.com/magicbug/Cloudlog/commit/61dbf5595c93a89ade1770a4107b68210a3c30d5)
